src/yang/ietf-bgp-rib.yang

```diff
@@ -117,23 +117,40 @@ submodule ietf-bgp-rib {
         description
           "BGP attribute defining the origin of the path
            information.";
+        reference
+          "RFC 4271: Section 5.1.1.";
       }
-      leaf atomic-aggregate {
-        type boolean;
+      container as-path {
         description
-          "BGP attribute indicating that the prefix is an atomic
-           aggregate; i.e., the peer selected is a less specific
-           route without selecting a more specific route that is
-           subsumed by it.";
+          "Enclosing container for the list of AS path segments.
+
+           In the Adj-RIB-In or Adj-RIB-Out, this list should show
+           the received or sent AS_PATH, respectively.  For
+           example, if the local router is not 4-byte capable, this
+           value should consist of 2-octet ASNs or the AS_TRANS
+           (AS 23456) values received or sent in BGP updates.
+
+           In the Loc-RIB, this list should reflect the effective
+           AS path for the route, e.g., a 4-octet value if the
+           local router is 4-octet capable.";
         reference
-          "RFC 4271: Section 5.1.6.";
+          "RFC 4271 - A Border Gateway Protocol 4 (BGP-4)
```

Comment:
This is more of a nit. But to be consistent do we want to follow the pattern of

"RFC XXXX: <Title of the RFC> [Section number].";

Reply:
This one is simply a move of one line to another.  Multiple files need cleanup on this point.  Let me open another issue for that and someone can pick it up as cleanup.

Reply:
Added Issue 263